packages/amplify-gen2-codegen/API.md

```diff
@@ -140,8 +186,10 @@ export type StorageTriggerEvent = 'onDelete' | 'onUpload';
 // @public (undocumented)
 export type UserPoolMfaConfig = 'OFF' | 'ON' | 'OPTIONAL';
 
-// @public (undocumented)
-export type UserPoolOverrides = Partial<Record<PasswordPolicyPath, string | number | boolean>>;
+// Warnings were encountered during analysis:
+//
+// src/auth/source_builder.ts:73:3 - (ae-forgotten-export) The symbol "MetadataOptions" needs to be exported by the entry point index.d.ts
+// src/auth/source_builder.ts:86:3 - (ae-forgotten-export) The symbol "OidcEndPoints" needs to be exported by the entry point index.d.ts
```

Comment:
this should be addressed.

Reply:
Resolved - [60d2ac9](https://github.com/aws-amplify/amplify-cli/pull/13913/commits/60d2ac94878b76ac7627ea01c51058cbc42324ef)